openpype/hosts/fusion/plugins/publish/validate_expected_frames_existence.py

```diff
@@ -23,25 +23,15 @@ def get_invalid(cls, instance, non_existing_frames=None):
         if non_existing_frames is None:
             non_existing_frames = []
 
-        if instance.data.get("render_target") == "frames":
-            tool = instance[0]
+        if "render.frames" in instance.data.get("families", []):
```

Comment:
This plugin now operates on `render.frames` instances. Isn't this ALWAYS true for those instances?